newproject2.php

```diff
@@ -1,11 +1,23 @@
 <?php
-$db = new PDO('mysql:host=db; dbname=Timelord_2021-02-08.sql', 'root', "password");
-$query = $db->prepare("query;");
+$db = new PDO('mysql:host=db; dbname=Timelord', 'root', "password");
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare('SELECT * FROM forgallifrey');
 $query->execute();
-$firstResult = $query->fetchAll();
-var_dump($firstResult);
+$forgallifreyArray = $query->fetchAll();
+//var_dump($forgallifreyArray);
+
+
+foreach ($forgallifreyArray as $item){
+   foreach($item as $key => $value){
+       echo $key.": ".$value."<br>";
+       if ($key==="No. of Adds"){
+           echo "</p><p>";
+       }
+   }
+}
 
 ?>
+
 <html lang="en-GB">
 
 <head>
```

Comment:
<head> needs a closing tag. I didn't spot one

Reply:
Not sure what you mean here? The html does have a closing tag at the end of the doc and the header has one soon after?